search/result.cpp

```diff
@@ -64,6 +66,24 @@ uint32_t Result::GetFeatureType() const
   return m_featureType;
 }
 
+std::string Result::GetReadableFeatureType() const
+{
+  ASSERT_EQUAL(m_resultType, Type::Feature, ());
+  return classif().GetReadableObjectName(m_featureType);
+}
+
+std::string Result::GetLocalizedFeatureType() const
+{
+  ASSERT_EQUAL(m_resultType, Type::Feature, ());
+  return platform::GetLocalizedTypeName(GetReadableFeatureType());
+}
```

Comment:
1. No need to create a function if it's used only once.
2. No need to call ASSERT twice.

```suggestion
std::string Result::GetLocalizedFeatureType() const
{
  ASSERT_EQUAL(m_resultType, Type::Feature, ());
  return platform::GetLocalizedTypeName(classif().GetReadableObjectName(m_featureType));
}
```

Reply:
Inlined